ee/query-service/model/license.go

```diff
@@ -104,3 +104,43 @@ type SubscriptionServerResp struct {
 	Status string   `json:"status"`
 	Data   Licenses `json:"data"`
 }
+
```

Comment:
Can we remove some of the models here?

Reply:
we are using all of the models for v3 here and will be required. any specific model are you referring here ?